phoenix-core/src/main/java/org/apache/phoenix/mapreduce/index/IndexUpgradeTool.java

```diff
@@ -652,51 +652,15 @@ private int startIndexRebuilds(Connection conn,
                     (GLOBAL_INDEX_ID.equals(tenantId)?"":"_"+tenantId) +"_"
                     + UUID.randomUUID().toString();
             String[] args = getIndexToolArgValues(schema, baseTable, indexName, outFile, tenantId);
-            Connection newConnection = conn;
-            Connection tenantConnection = null;
             try {
                 LOGGER.info("Rebuilding index: " + StringUtils.join( args,","));
                 if (!dryRun) {
-                    // If the index is in DISABLED state, indexTool will fail.
-                    // First to ALTER REBUILD ASYNC.
```

Comment:
this is now taken care of within the index tool?

Reply:
I am doing PHOENIX-5512 right now to take care of this